### PR TITLE
Loginform

### PR DESCRIPTION
**Pull request by Oleg-Decik**

### CODE REVIEW

File: src/main/java/test/OlegLoginTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import org.openqa.selenium.By;
+import org.testng.annotations.Test;
+
+public class OlegLoginTest {
+	@Test
+	public void testLoginForm(){
+		WebDriver webDriver = new WebDriver ();
+		UnlogUserInfo unlogUserInfo = new UnlogUserInfo ();
+		unlogUserInfo.clickSignIn ();
+		LoginForm loginForm = new LoginForm (  );
```
Comment (ostret) [inline src/main/java/test/OlegLoginTest.java:16]: why are we creating one webdriver in test method and then create another in initLogin?

File: src/main/java/test/OlegLoginTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		String expected = "Admin admin";
+		String actual = WebDriver.driver.findElement ( By.cssSelector ( "#_desktop_user_info > div > a.account > span" ) ).getText ();
+		assertEquals ( expected, actual );
```
Comment (ostret) [inline src/main/java/test/OlegLoginTest.java:19]: you know that in testng it's vice versa

https://static.javadoc.io/org.testng/testng/6.8.17/org/testng/Assert.html#assertEquals(boolean,%20boolean)

File: src/main/java/test/OlegLoginTest.java (hunk 1 above)
Comment (Oleg-Decik) [reply src/main/java/test/OlegLoginTest.java:16]: it was just for test, i will correct it right now

File: src/main/java/test/OlegLoginTest.java (hunk 2 above)
Comment (Oleg-Decik) [reply src/main/java/test/OlegLoginTest.java:19]: oh yes as i said i just test it, it will be correct soon)

**Pull request closed without merging**